### PR TITLE
New version: Microsoft.Edge.Beta version 99.0.1150.11

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/45640)

### CODE REVIEW

**Comment by jedieaston** [comment]
Let me see if my PR works...

File: manifests/m/Microsoft/Edge/Beta/99.0.1150.11/Microsoft.Edge.Beta.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  InstallerSha256: B1D575920A85B7F82ABE06B47E0195AD920056EBB84ACEAEA6CD7F1DD9597DE5
+  ProductCode: '{AA205979-6E9C-30BA-BD28-2F66FE821101}'
+- Architecture: x64
+  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/b4a5f407-413d-4053-8156-47f62d15032f/MicrosoftEdgeBetaEnterpriseX86.msi
+  InstallerSha256: 3E734F3858D98E06C6ACAD68A6199A5010F46B8926B57DC5B86E557EB528FF54
+  ProductCode: '{478ECA66-D387-3054-ADE0-C0B276045642}'
+- Architecture: x86
+  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/534e2dbd-69d7-4b04-8158-b38fb4a8f316/MicrosoftEdgeBetaEnterpriseARM64.msi
+  InstallerSha256: 49B7A55BF98B7E2CB41718A481F7F88CEFEB87E7D702B9669425AC56045C6B64
+  ProductCode: '{272A0540-2F28-39C6-B081-69331D093529}'
```
Comment (jedieaston) [inline manifests/m/Microsoft/Edge/Beta/99.0.1150.11/Microsoft.Edge.Beta.installer.yaml:30]: The architectures are confused!

**Review by jedieaston — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/Edge/Beta/99.0.1150.11/Microsoft.Edge.Beta.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- http
+- https
+FileExtensions:
+- crx
+- htm
+- html
+- pdf
+- url
+Installers:
+- Architecture: arm64
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/Edge/Beta/99.0.1150.11/Microsoft.Edge.Beta.installer.yaml:19]:
```suggestion
- Architecture: x64
```

File: manifests/m/Microsoft/Edge/Beta/99.0.1150.11/Microsoft.Edge.Beta.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/3aeaadab-046d-4143-90f6-7943e9ac9bdc/MicrosoftEdgeBetaEnterpriseX64.msi
+  InstallerSha256: B1D575920A85B7F82ABE06B47E0195AD920056EBB84ACEAEA6CD7F1DD9597DE5
+  ProductCode: '{AA205979-6E9C-30BA-BD28-2F66FE821101}'
+- Architecture: x64
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/Edge/Beta/99.0.1150.11/Microsoft.Edge.Beta.installer.yaml:23]:
```suggestion
- Architecture: x86
```

File: manifests/m/Microsoft/Edge/Beta/99.0.1150.11/Microsoft.Edge.Beta.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  InstallerUrl: https://msedge.sf.dl.delivery.mp.microsoft.com/filestreamingservice/files/b4a5f407-413d-4053-8156-47f62d15032f/MicrosoftEdgeBetaEnterpriseX86.msi
+  InstallerSha256: 3E734F3858D98E06C6ACAD68A6199A5010F46B8926B57DC5B86E557EB528FF54
+  ProductCode: '{478ECA66-D387-3054-ADE0-C0B276045642}'
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/Edge/Beta/99.0.1150.11/Microsoft.Edge.Beta.installer.yaml:27]:
```suggestion
- Architecture: arm64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**